app/scripts/modules/core/application/config/serializeApplicationSection.directive.html

```diff
@@ -0,0 +1,18 @@
+<div class="row">
+  <div class="col-md-7">
+    <h4>Serialize Application</h4>
+    <div class="panel panel-default">
+      <div class="panel-body">
+        <p>
+          Serializing the application will save the current state of the infrastructure using Terraform in a bucket. <br>
```

Comment:
Or:

Serializing the application will save the current state of the
infrastructure to a storage bucket using Terraform's config language. (with
the link)

On Fri, Jul 22, 2016 at 5:42 PM Lars Wander <notifications@github.com>
wrote:

> In
> app/scripts/modules/core/application/config/serializeApplicationSection.directive.html
> <https://github.com/spinnaker/deck/pull/2468#discussion_r71950311>:
>
> > @@ -0,0 +1,18 @@
> > +<div class="row">
> > +  <div class="col-md-7">
> > +    <h4>Serialize Application</h4>
> > +    <div class="panel panel-default">
> > +      <div class="panel-body">
> > +        <p>
> > +          Serializing the application will save the current state of the infrastructure using Terraform in a bucket. <br>
>
> Maybe word to "... of the infrastructure using Terraform's config language
> <https://www.terraform.io/docs/configuration/index.html>. in a bucket."
> (making sure to include that link).
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/spinnaker/deck/pull/2468/files/71531b8c76e2009271c4859f43a0632fc7a713e7#r71950311>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIC_1hKCPgP6BeFgA1jD7QfwpEMnuc-Gks5qYTk2gaJpZM4JTF3Z>
> .
>